interfaces/PA-App/src/app/user-menu/user-menu.component.html

```diff
@@ -11,7 +11,12 @@
     <ion-item button="true" disabled>
       {{ 'account.change-password'|translate}}
     </ion-item>
-    <ion-item button="true" disabled>
+    <ion-item button="true"
+              [disabled]="!isLoggedIn"
+              (click)="deletePrompt()"
+              detail="true"
+              detail-icon="log-out"
```

Comment:
I would think there is a more destructive icon available for this. :D 

(But for the user it will end up at the same place, 'outside'. So fine. :) )